app/views/organizations/home/index.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="row d-flex align-items-center">
       <div class="col-xxl-5  col-xl-6 col-lg-6 col-12">
         <div>
-          <h1 class="display-2 fw-bold mb-3">Rescue pets near you at <u class="text-warning"><span class="text-primary"> <%= Current.organization.name %></span></u></h1>
+          <h1 class="display-2 fw-bold mb-3">Rescue pets in <%= Current.organization&.location&.city_town %> at <u class="text-warning"><span class="text-primary"> <%= Current.organization.name %></span></u></h1>
```

Comment:
A quick thought: by adjusting the controller inheritance, both organization and location should always be available, eliminating the need for &